QRCodeHere/Watermark+ViewModifier.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Watermark+ViewModifier.swift
+//  QRCodeHere
+//
+//  Created by Abdullah Alhaider on 05/07/2021.
+//  Copyright © 2021 Abdullah Alhaider. All rights reserved.
+//
+
+import SwiftUI
+
+struct Watermark: ViewModifier {
+    var text: String?
+    
+    func body(content: Content) -> some View {
+        if let text = text {
+            ZStack {
+                Color.white
+                VStack {
+                    content
+                        .padding(.top, 20)
```

Comment:
I suggest removing this padding to prevent the content above the image from being pushed up.
```suggestion
```

Reply:
This is needed to make the UI consistent

Reply:
Done